src/Components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import './NewTodo.css';
+import cx from 'classnames';
+import { newTodoPropTypes } from '../PropTypes/PropTypes';
+
+class NewTodo extends React.Component {
+  state = {
+    users: [...this.props.users],
+    id: 3,
+    inputTitle: '',
+    inputUser: '',
+    errorTitle: null,
+    errorUser: null,
+  }
+
+  handleInputTitleChange = ({ target }) => {
+    this.setState({
+      inputTitle: target.value,
+      errorTitle: null,
+    });
+  }
+
+  handleUserChange = ({ target }) => {
+    this.setState({
+      inputUser: target.value,
+      errorUser: null,
+    });
+  }
+
+  handleButtonSubmit = (event) => {
+    event.preventDefault();
+    const {
+      inputTitle, inputUser, id, users,
+    } = this.state;
+    const { onAdd } = this.props;
```

Comment:
не очень понятно что и куда добавляется
```suggestion
    const { addNewTodo } = this.props;
```